index.html

```diff
@@ -30,5 +30,94 @@ <h1 class="text-primary-1 ff-crete line-height-1 ">Hey There. <br> I'm Tomi</h1>
            <li> <a href="#"  class="btn-contact linkedin"> </a> </li> 
         </ul>
     </main>
+    <section class="container work-section grid bg-light">
+        <h2 class="ff-crete mg-8 center">My Recent Works</h2>
+        <div class="featured">
+            <img src="assets/img/img-placeholder.png" alt="">
+            <h3 class="mg-1 ff-crete" >Multi-Post Stories</h3>
+            <p class="ff-inter"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text. </p>
+            <ul class="tag tag-feature mg-1 ff-inter flex">
+                <li> <a href="#"> css </a> </li>
+                <li> <a href="#"> html </a> </li>
+                <li> <a href="#"> bootstrap </a> </li>
+                <li> <a href="#"> Ruby </a> </li>
```

Comment:
- [ ] Kindly ensure that all the li tags are on a single row, please check the screenshot above and make your design look more like the Figma design 👍🏿